edx_credentials_themes/__init__.py

```diff
@@ -1 +1 @@
-__version__ = '0.1.4'
+__version__ = '0.1.6'
```

Comment:
0.1.5

Reply:
0.1.5 seems to already exist (there's a tag for it). I think someone may have just forgotten to bump the version.